scripts/e2e_test/e2e_test.rb

```diff
@@ -21,7 +21,7 @@ class TestRunner
 
   NUM_RUNS_TO_KEEP = 5
   PROJECT_ROOT = File.expand_path("#{__dir__}/../..")
-  TEST_ROOT = File.expand_path("#{PROJECT_ROOT}/../mailing_e2e_tests")
+  TEST_ROOT = File.expand_path("#{PROJECT_ROOT}/tmp/e2e_test")
```

Comment:
moved this to a local tmp dir just to not create a random folder outside of this project.

Reply:
it does that so that it doesn't accidentally use anything from the project's node_modules

Reply:
+1 you will want to move this outside of the project directory

Reply:
does github not let you do that?

Reply:
the other option would be nuking "#{PROJECT_ROOT}/node_modules" but that would probably have other reprecussions